odata-url-conventions/4 Resource Path.md

```diff
@@ -979,18 +980,16 @@ Example ##ex: passing multiple system query options in the request body
 POST http://host/service/People/$query
 Content-Type: application/x-www-form-urlencoded
 
-%24filter=[FirstName,LastName]+in+[["John","Doe"],["Jane","Smith"]]&
-%24select=FirstName,LastName
+%24filter=LastName+eq+%27P%26G%27&%24select=FirstName,LastName
 ```
 This POST request would result from submitting the HTML form
 ```html
 <form method="post" action="http://host/service/People/$query"
       enctype="application/x-www-form-urlencoded">
-  <input name="$filter"
-    value='[FirstName,LastName] in [["John","Doe"],["Jane","Smith"]]'>
+  <input name="$filter" value="LastName eq 'P&G'">
   <input name="$select" value="FirstName,LastName">
 </form>
 ```
-Note `enctype="text/plain"` would not encode the spaces and would produce a newline
-instead of the `&` demanded by the OData syntax.
+Note `enctype="text/plain"` would not encode the spaces and ampersand and would
+separate the options with a newline instead of the `&` demanded by the OData syntax.
```

Comment:
```suggestion
separate the options with a newline instead of the `&`.
```
I think the ampersand separator is required by `application/x-www-form-urlencoded`

Reply:
The OData syntax requires that system query options are separated by a `&`. (There cannot be newlines in a URL.)

Reply:
True, for URLs.

In requests bodies of content-type `application/x-www-form-urlencoded` it's the definition of that content type that requires the ampersands, just as you explain that `text/plain` requires newline as a separator.

Nothing OData-specific here.

Reply:
A `POST ~/$query` with `Content-Type: text/plain` requires an `&` as separator between the system query options, because the current spec says:
> The request body MUST use the content-type text/plain. It contains the query portion of the URL and MUST use the same percent-encoding as in URLs (especially: no spaces, tabs, or line breaks allowed) and MUST follow the syntax rules described in chapter Query Options.

This is OData-specific and does not align with the behavior of text/plain HTML forms.

Reply:
Too bad: can we also allow the default behavior of text/plain?

Reply:
But besides that, the x-www ampersand thingy has nothing to do with OData, so better remove that half sentence.